### DISCUSSION TITLE
Build Time Error on mvn dependency:go-offline

### DISCUSSION THREAD

**Discussion by tricktron** [dbeaver/dbeaver#21275] [DBeaver Development] [answered]
### Description

I wanted to download all dependencies to build dbeaver on a machine without internet.

However, downloading all dependencies with jdk 17 and maven 3.9.4 with
`mvn dependency:go-offline -Dmaven.repo.local=./.m2` fails with:

Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.6.0:go-offline (default-cli) 
on project org.jkiss.dbeaver.rcp.feature: Execution default-cli of goal org.apache.maven.plugins:maven-dependency-plugin:3.6.0:go-offline failed: 
java.lang.RuntimeException: Equinox runtime location is missing or invalid.

### DBeaver Version

23.2.0

### Operating System

MacOS 13.5.2

### Steps to reproduce

1. `cd features/org.jkiss.dbeaver.rcp.feature && mvn dependency:go-offline -Dmaven.repo.local=./.m2` or just `mvn dependency:go-offline -Dmaven.repo.local=./.m2`.

**Comment by serjiokov** [comment, marked as answer]
Hi, to resolve issue, you can try to download all require dependency on your local mashine manually (dependency specified in root pom.xml) next retarget pom.xml to your local pathes or preinstall dependency to your local m2-repository.
